db/migrate/20150727042746_create_gems_adoptions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGemsAdoptions < ActiveRecord::Migration
+  def change
+    create_table :gems_adoptions do |t|
+      t.integer :user_id
+      t.string :gem
+      t.text :description
```

Comment:
I think we should be checking that these columns are not `nil`, we probably don't records in the database with at least `user_id` and `gem` being `null`, do you agree?